less_slow.cpp

```diff
@@ -525,7 +552,7 @@ static void sorting_with_openmp(bm::State &state) {
 
 #pragma omp parallel for
         // Sort each chunk in parallel
-        for (std::size_t i = 0; i < chunks; i++) {
+        for (int64_t i = 0; i < chunks; i++) {
```

Comment:
Why switch to to `int64_t` here and in the lines below?

Reply:
I mentioned it in the notes, but, at least on msvc, openmp only allows signed integers as the parallel for loop index/iterator.